__tests__/test.js

```diff
@@ -75,21 +74,52 @@ describe("GET /api/articles ", () => {
     return request(app)
       .get("/api/articles")
       .then((res) => {
-        expect(res.status).toBe(200);
+        expect(200);
         expect(res.body.length).toBeGreaterThan(0);
-        expect(res.body).toBeSorted("comments.created_at");
-        expect(res.body).toBeSorted({ descending: true });
-        res.body.forEach((topic) => {
-          expect(topic).toHaveProperty("title");
-          expect(topic).toHaveProperty("article_id");
-          expect(topic).toHaveProperty("author");
-          expect(topic).not.toHaveProperty("body");
-          expect(topic).toHaveProperty("topic");
-          expect(topic).toHaveProperty("created_at");
-          expect(topic).toHaveProperty("votes");
-          expect(topic).toHaveProperty("article_img_url");
-          expect(topic).toHaveProperty("comment_count");
+        expect(res.body).toBeSortedBy("created_at", { descending: true });
+        res.body.forEach((article) => {
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("author");
+          expect(article).not.toHaveProperty("body");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments ", () => {
+  test(".../:article_id/comments returns all comments for given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then((res) => {
+        expect(200);
+        expect(res.body.length).toBeGreaterThan(0);
+        expect(res.body).toBeSortedBy("created_at", {
+          descending: true,
         });
+        const expectedCommentStructure = {
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Nice use of `toMatchObject` here, though you could be more specific about what `article_id` to expect